cfg/rippled-example.cfg

```diff
@@ -1629,10 +1629,10 @@ ip = 127.0.0.1
 admin = 127.0.0.1
 protocol = ws
 
-#[port_grpc]
-#port = 50051
-#ip = 0.0.0.0
-#secure_gateway = 127.0.0.1
+[port_grpc]
+port = 50051
+ip = 0.0.0.0
+secure_gateway = 127.0.0.1
```

Comment:
This is just to make it easier for people to run clio, since clio uses this port to talk to rippled. Though now that I'm thinking about it, it might be best to change the IP to `127.0.0.1`, so that way the port will only listen locally by default. If someone is running clio remotely, they'll need to modify their config anyways.